tests/infra/AutocompleteServiceImpl.ts

```diff
@@ -306,3 +307,64 @@ describe('additionalQueryParams are passed through', () => {
     }));
   });
 });
+
+describe('sandbox endpoints work as expected', () => {
```

Comment:
Personally, I lean to 1.

Reply:
If we remove the endpoint defaulting from within the services, we'll need to an Object.assign into either AnswersCore or provideCore for user defined endpoints and default endpoints. I still would want coverage for that, which we don't have any for now, but it would be pretty simple.
I thought about just checking the member variables on the services, but they're private. I don't think we want to make them public just for testing purposes.

Reply:
tldr; I like option 1 and making the services dumber

Reply:
Oh I didn't see your "Personally, I lean to 1." comment, thanks github